chain/client/src/debug.rs

```diff
@@ -618,6 +613,12 @@ impl ClientActor {
             shards: self.client.runtime_adapter.num_shards(&head.epoch_id).unwrap_or_default(),
             approval_history: self.client.doomslug.get_approval_history(),
             production: productions,
+            banned_chunk_producers: self
+                .client
+                .do_not_include_chunks_from
```

Comment:
Could you sort the vec here? I think it makes the display easier to see, especially if this has invalid chunk producers in many epochs.

Reply:
Actually maybe use something like Vec<(EpochId, Vec<AccountId>)>

Reply:
Done. Thanks!